MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift

```diff
@@ -11,32 +11,96 @@
  
  In the following questions, where we use the term `variable`, we are not specifying whether or not it's a variable that is declared with `var` or a constant which is declared with `let`. We are using this general term, leaving it up to you to decipher whether or not you need to use `var` or `let` within the solution.
  */
-/*: question1
- ### 1. Create a function where the two arguments represent different bank account (one of those bank accounts is yours). What type should these arguments be if we are to then perform some math operations on them? Setup a conditional that will add 10 to your funds and minus 10 from the other funds (bank account) if the other account won't go negative if we were to take 10 dollars from it.
- */
+
+
+// Given....
+let x = 5.0
+let y = 12
+let a = 321
+let b = 32
+
+
+
+/*: Question 1
+### 1. Print the result of a greater than or equal to b
+*/
 // write your code here
 
 
+/*: Question 2
+### 2. Print the result of a mod b equals zero
```

Comment:
Slight change here. I think I would like to this to read as...

Print the result of a mod b is equal to zero